mne/channels/interpolation.py

```diff
@@ -132,11 +132,18 @@ def _interpolate_bads_eeg(inst, origin, verbose=None):
     ----------
     inst : mne.io.Raw, mne.Epochs or mne.Evoked
         The data to interpolate. Must be preloaded.
+    origin : array-like, shape (3,) | str
+        Origin of the sphere in the head coordinate frame and in meters.
+        Can be ``'auto'`` (default), which means a head-digitization-based
+        origin fit.
+    exclude : list | tuple
+        The channels to exclude from interpolation. If excluded a bad
+        channel will stay bad.
```

Comment:
I would rather kill this docstring entirely rather than have to continue to update it, I think it actually reduces maintenance burden